google-cloud-video-stitcher/samples/Gemfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+
+master = ENV["GOOGLE_CLOUD_SAMPLES_TEST"] == "master"
+
+gem "google-cloud-video-stitcher", path: master ? "../../google-cloud-video-stitcher" : nil
+
+group :test do
+  gem "google-style", "~> 1.25.1"
+  gem "minitest", "~> 5.14"
```

Comment:
Would you mind using `"google-style", "~> 1.26.1"` and `"minitest", "~> 5.16"`? We're trying to get everything updated to those versions.

Reply:
Done